### PR TITLE
Update metadata.json

### PR DESCRIPTION
**Pull request by GreenScreen410**
Change old messenger logo

### CODE REVIEW

**Review by ririxidev — CHANGES_REQUESTED** [review]
version must be bumped - logo must be updated on discord developers portal.

File: websites/M/Messenger/dist/metadata.json [hunk 1]
```diff
@@ -10,12 +10,12 @@
   },
   "url": "www.messenger.com",
   "version": "1.1.5",
```
Comment (Bas950) [inline websites/M/Messenger/dist/metadata.json:12]:
```suggestion
  "version": "1.1.6",
```

**Comment by Bas950** [comment]
> version must be bumped - logo must be updated on discord developers portal.

image
Done on dev portal.

**Comment by ririxidev** [comment]
No response from pull request author :/

**Pull request closed without merging**